src/Microsoft.AspNet.FileProviders.Composite/CompositeFileProvider.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNet.FileProviders
+{
+    /// <summary>
+    /// Looks up files using a list of <see cref="IFileProvider"/>.
+    /// </summary>
+    public class CompositeFileProvider : IFileProvider
+    {
+        private readonly IFileProvider[] _fileProviders;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CompositeFileProvider" /> class using a list of file provider.
+        /// </summary>
+        /// <param name="fileProviders"></param>
+        public CompositeFileProvider(params IFileProvider[] fileProviders)
```

Comment:
Did we settle on using `params`? I thought @davidfowl wanted to use an `IEnumerable` here?